ign_rviz_plugins/include/ignition/rviz/plugins/LaserScanDisplay.hpp

```diff
@@ -46,25 +55,62 @@ class LaserScanDisplay : public MessageDisplay<sensor_msgs::msg::LaserScan>
   ~LaserScanDisplay();
 
   // Documentation Inherited
-  void initialize(rclcpp::Node::SharedPtr);
+  void LoadConfig(const tinyxml2::XMLElement * /*_pluginElem*/);
 
   // Documentation Inherited
-  void callback(const sensor_msgs::msg::LaserScan::SharedPtr);
+  void initialize(rclcpp::Node::SharedPtr _node);
+
+  // Documentation Inherited
+  void callback(const sensor_msgs::msg::LaserScan::SharedPtr _msg);
+
+  // Documentation inherited
+  void setTopic(std::string topic_name);
 
   // Documentation inherited
-  void setTopic(std::string);
+  void subscribe();
+
+  // Documentation inherited
+  void reset();
```

Comment:
Are you overriding some of this methods?

```suggestion
  void reset() override;
```